CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# 45.0
+- Add support for 'flutter pub outdated' (#4444)
+- Improve how we calculate when to summarize flutter errors (#4447)
+- Fix an issue with discovering hyperlinks in test consoles (#4443)
```

Comment:
These first three items (and the new supported platforms) would be good fodder for the blog.